Makefile

```diff
@@ -7,6 +7,18 @@ cppflags := -DDATADIR=$(DATADIR)
 
 sources := $(wildcard src/brogue/*.c) $(addprefix src/platform/,main.c platformdependent.c)
 
+ifeq ($(shell git tag --points-at HEAD),)
```

Comment:
`shell` expands to empty string if the command is not available (no output is written), so this gives a false positive when Git is not available.

Reply:
That is a good catch, thank you.  Per the other thread maybe I can guard (or eliminate) uses of `git` directly with CI-supplied env vars.